README.rst

```diff
@@ -1,16 +1,26 @@
-=========
-intbitset
-=========
+===========
+ intbitset
+===========
+
+.. image:: https://img.shields.io/travis/inveniosoftware/intbitset.svg
+        :target: https://travis-ci.org/inveniosoftware/intbitset
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/intbitset.svg
+        :target: https://coveralls.io/r/inveniosoftware/intbitset
```

Comment:
@kaplun it should be possible (see http://blog.behnel.de/posts/coverage-analysis-for-cython-modules.html)